hw1/task04/test_task04.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from task04 import check_sum_of_four
+
+all_zero_x5 = [0] * 5
+empty = []
+max_length = list(range(1000))
+
+
+@pytest.mark.parametrize(
+    ("value", "expected_result"),
+    [
+        ((all_zero_x5, all_zero_x5, all_zero_x5, all_zero_x5), 625),
```

Comment:
Reusing variables is good practice, but sometimes it is better to avoid it in tests and just hardcode values.

Reply:
I don't understand it in this case, but I will trust you!